lms/djangoapps/discussion_api/api.py

```diff
@@ -244,6 +254,11 @@ def get_thread_list(request, course_key, page, page_size, topic_id_list=None, te
     topic_id_list: The list of topic_ids to get the discussion threads for
     text_search A text search query string to match
     following: If true, retrieve only threads the requester is following
+    order_by: The key in which to sort the threads by. The only values are
+        "last_activity_at", "comment_count", and "vote_count". The default is
+        "last_activity_at".
```

Comment:
Why aren't the defaults set above in the function definition? order_by="last_activity_at" and order_direction="desc"?